stable-profiles/0.9.52/firejailed-tor-browser.profile

```diff
@@ -0,0 +1,117 @@
+# Firejail profile for firejailed-tor-browser
+# This file is overwritten after every install/update
+# Persistent local customizations
+include ${HOME}/.config/firejail/firejailed-tor-browser.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+# Note: PluggableTransports didn't work with this profile
+
+noblacklist ${HOME}/.firejailed-tor-browser
+noblacklist ${HOME}/.config/firejail
```

Comment:
Why does it need access to it?

Reply:
Ah! You area correct., this is not needed. Thank you for the correction.

Reply:
On second thought, there are similar statements included in the default `/etc/firejail/torbrowser-launcher.profile` which I initially used as a template for creating my own .profile for my specific environment.